MAUI/TreeMap/tooltip.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: post
+title: Tooltip in .NET MAUI TreeMap Control | Syncfusion
+description: Learn here all about how to customize the appearance of tooltip in Syncfusion .NET MAUI TreeMap control and more.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+
+# Tooltip in .NET MAUI TreeMap (SfTreeMap)
+
+The tooltip provides information when hovering over or selecting items in the interactive TreeMap, displaying details about the values of each item.
+
+## Enable tooltip
+
+To enable tooltip for the TreeMap control, utilize the `ShowToolTip` property of `SfTreeMap.` By default, the value of `ShowToolTip` is set to `false.` To provide users with additional information or context about specific treemap items, simply set this property to `true.`
```

Comment:
To enable the tooltip for the TreeMap control, utilize the `ShowToolTip` property of `SfTreeMap.` By default, the value of `ShowToolTip` is set to `false.` To provide users with additional information or context about specific treemap items, simply set this property to `true.`

Reply:
Changed content